### PR TITLE
applications: nrf5340_audio: Remove dependencies from error_handler

### PR DESCRIPTION
**Pull request by erikrobstad**
The center LED is now controlled
by the GPIO driver directly

OCT-2463

### CODE REVIEW

**Comment by erikrobstad** [comment]
One question: 
Should we set the LED if the config is release (if CONFIG_DEBUG=n)?
In this case the system will reboot right after

**Comment by alexsven** [comment]
> One question: Should we set the LED if the config is release (if CONFIG_DEBUG=n)? In this case the system will reboot right after

Don't think that should be necessary no

File: applications/nrf5340_audio/src/utils/error_handler.c [hunk 1]
```diff
@@ -8,48 +8,43 @@
...
-#if defined(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP)
-	(void)led_on(LED_APP_RGB, LED_COLOR_RED);
-#elif defined(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUNET)
-	(void)led_on(LED_NET_RGB, LED_COLOR_RED);
-#endif /* defined(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP) */
-}
+/* nRF5340 Audio DK center RGB LED */
+static const struct gpio_dt_spec center_led_r = GPIO_DT_SPEC_GET(DT_NODELABEL(rgb1_red), gpios);
+static const struct gpio_dt_spec center_led_g = GPIO_DT_SPEC_GET(DT_NODELABEL(rgb1_green), gpios);
+static const struct gpio_dt_spec center_led_b = GPIO_DT_SPEC_GET(DT_NODELABEL(rgb1_blue), gpios);
```
Comment (alexsven) [inline applications/nrf5340_audio/src/utils/error_handler.c:20]: This is now tied very heavily to our board. If the pin/pins were passed in with an init function this file could reside in NCS instead.

**Review by alexsven — CHANGES_REQUESTED** [review]

File: applications/nrf5340_audio/src/utils/error_handler.c (hunk 1 above)
Comment (koffes) [reply applications/nrf5340_audio/src/utils/error_handler.c:20]: That is possible, though, the system may fail before we have a chance to init the error handler.

**Review by koffes — CHANGES_REQUESTED** [review]

**Comment by erikrobstad** [comment]
Changed so that the center LED is only used if the board is Audio DK.
Also changed from gpio_pin_set_dt() to gpio_pin_configure_dt(), so that the error handler should work even if the LEDs are not configured beforehand.

**Review by alexsven — APPROVED** [review]

**Review by gWacey — APPROVED** [review]

**Pull request merged**